controllers/articles.controller.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const app = express();
+const index = require('../db/data/test-data/index')
+const { fetchArticles } = require('../models/articles.model')
+
+app.use(express.json());
```

Comment:
^^ what does this code do? It isn't relevant for get requests, and also should only appear in the app.js when it is relevant